src/Devito.jl

```diff
@@ -7,23 +7,34 @@ const sympy = PyNULL()
 const devito = PyNULL()
 const devitopro = PyNULL()
 const seismic = PyNULL()
+const utils = PyNULL()
 
-has_devitopro() = haskey(PyCall.Conda._installed_packages_dict(), "devitopro")
+has_devitopro() = devitopro != devito
 
 function __init__()
     try
         copy!(numpy, pyimport("numpy"))
         copy!(sympy, pyimport("sympy"))
         copy!(devito, pyimport("devito"))
-        copy!(devitopro, (has_devitopro() ? pyimport("devitopro") : pyimport("devito")))
+        try
```

Comment:
That's a "personal" prefernce to just check at init if devitopro is available then have `has_devitopro` just check if it's there